app/src/main/java/org/mozilla/fenix/experiments/NimbusFeatures.kt

```diff
@@ -0,0 +1,89 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.experiments
+
+import android.content.Context
+import android.os.StrictMode
+import org.mozilla.experiments.nimbus.mapKeysAsEnums
+import org.mozilla.fenix.ext.components
+import org.mozilla.fenix.ext.getVariables
+
+class NimbusFeatures(
+    private val context: Context,
+) {
+
+    val homeScreen : HomeScreenFeatures by lazy  {
+        HomeScreenFeatures(context)
+    }
+
+    class HomeScreenFeatures(private val context: Context) {
+        /**
+         * `FeatureId.HOME_PAGE` feature; the complete JSON, is shown here:
+         *
+         * ```json
+         * {
+         *     "sections-enabled": {
+         *         "topSites": true,
+         *         "recentlySaved": false,
+         *         "jumpBackIn": false,
+         *         "pocket": false,
+         *         "recentExplorations": false
+         *     }
+         * }
+         * ```
+         */
+
+        /**
+         * This enum accompanies the `FeatureId.HOME_PAGE` feature.
+         *
+         * These names here should match the names of entries in the JSON.
+         */
+        @Suppress("EnumEntryName")
+        private enum class HomeScreenSection(val default: Boolean) {
+            topSites(true),
+            recentlySaved(false),
+            jumpBackIn(false),
+            pocket(false),
+            recentExplorations(false),
+        }
+
+        private val homeScreenFeatures: Map<HomeScreenSection, Boolean> by lazy {
+            val experiments =
+                context.components.strictMode.resetAfter(StrictMode.allowThreadDiskReads()) {
+                    context.components.analytics.experiments
+                }
```

Comment:
I don't think we should be opting out of `StrictMode` here.

Reply:
👍🏽